regression/tests/studio/test_studio_textbooks.py

```diff
@@ -0,0 +1,72 @@
+"""
+Test textbook page
+"""
+
+from bok_choy.web_app_test import WebAppTest
+from edxapp_acceptance.pages.studio.overview import CourseOutlinePage
+
+from regression.pages.studio.login_studio import StudioLogin
+from regression.pages.studio.studio_textbooks import TextbookPageExtended
+from regression.pages.studio.studio_home import DashboardPageExtended
+from regression.tests.helpers import LoginHelper, get_course_info
+
+
+class TextbookTest(WebAppTest):
+    """
+    Test textbooks.
+    """
+    def setUp(self):
+        super(TextbookTest, self).setUp()
+        self.login_page = StudioLogin(self.browser)
+        LoginHelper.login(self.login_page)
+        self.course_info = get_course_info()
+
+        self.studio_course_outline = CourseOutlinePage(
+            self.browser, self.course_info['org'], self.course_info['number'],
+            self.course_info['run'])
+
+        self.studio_home_page = DashboardPageExtended(self.browser)
+
+        self.textbook_page = TextbookPageExtended(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+
+        self.studio_home_page.visit()
+        self.studio_home_page.select_course(
+            'Manual Smoke Test Course 1 - Auto')
+        self.studio_course_outline.wait_for_page()
+
+        self.studio_home_page.q(css='.nav-item.nav-course-courseware').click()
```

Comment:
I think you should only do this kind of navigation testing in one test for the thing you are testing, not all of them. It can turn into a lot of time waste if we use it too much; we should only use it for the time that it's important to test the navigation (i.e., only one test per feature, or something like that).